libbeat/cmd/instance/beat_test.go

```diff
@@ -1,6 +1,6 @@
 // +build !integration
 
-package beat
+package instance
```

Comment:
Some more cleanup on instance or cmd package would be nice. But this was nothing I wanted to touch right now.